WcaOnRails/app/assets/stylesheets/registrations.scss

```diff
@@ -25,3 +26,22 @@ table.registrations-table {
   z-index: 1000; /* show over sticky table headers */
   bottom: calc(5px + #{$footer-height-px});
 }
+
+/* Styling bootstrap table sort */
+.fixed-table-loading {
+  display: none;
+}
+
+th {
+  &:focus { outline: none; }
+  .sortable {
+    &:hover { cursor: pointer; }
+    &:after {
+      font-family: FontAwesome;
+      content: '\f0dc';
```

Comment:
Because we're using font-awesome-sass, you can actually use the variables as defined in https://github.com/FortAwesome/font-awesome-sass/blob/master/assets/stylesheets/font-awesome/_variables.scss.